src/ape/managers/chain.py

```diff
@@ -188,11 +188,13 @@ def range(
     def poll_blocks(
         self,
         start: Optional[int] = None,
+        stop: Optional[int] = None,
         required_confirmations: Optional[int] = None,
     ) -> Iterator[BlockAPI]:
         """
         Poll new blocks. Optionally set a start block to include historical blocks.
-        **NOTE**: This is a daemon method; it does not terminate unless an exception occurrs.
+        **NOTE**: This is a daemon method; it does not terminate unless an exception occurrs
+        or a ``stop_block`` is given.
```

Comment:
Perhaps the input variables should be renamed, they don't match this

Reply:
Right now, they are copied from the built-in range: https://www.w3schools.com/python/ref_func_range.asp

Reply:
I mean that the input variable's name is `stop` but it says `stop_block` here